challenge_5/python/ajschrier/FindTheDifference.py

```diff
@@ -0,0 +1,21 @@
+from collections import Counter
+
+
+class Solution():
```

Comment:
There's no need to use a class here as you are effectively defining a standalone method. Just make it a module level method.